recipe/meta.yaml

```
@@ -24,97 +22,97 @@ source:
     folder: src/gpl
 
 {% set lc, vs, ve, prefix, parbuild = "\\", "$", "", "$PREFIX", "-j $CPU_COUNT" %}  # [unix]
-{% set lc, vs, ve, prefix, parbuild = "^", "%", "%", "%LIBRARY_PREFIX%", "" %}      # [win]
 build:
-  number: 1002
-  features:
-    - blas_{{ blas_variant }}  # [not win]
-  skip: true  # [win32 or (win and vc<14)]
+  number: 0
+  skip: true  # [win32 or win]
   script: |
     mkdir build
     cd build
-
-    cmake ..                                 {{ lc }}
-      -G"{{vs}}CMAKE_GENERATOR{{ve}}"        {{ lc }}
-      -DCMAKE_PREFIX_PATH="{{ prefix }}"     {{ lc }}
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
+      -DENABLE_TESTING=OFF                                    {{ lc }}
+      -DENABLE_COVERAGE=OFF                                   {{ lc }}
+      -DUSE_SWIG_DIRECTORS=ON                                 {{ lc }}
+      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [bla_vendor == 'Intel']
+      -DBLAS_LIBRARIES="{{ PREFIX }}/lib/libcblas{{ SHLIB_EXT }};{{ PREFIX }}/lib/libblas{{ SHLIB_EXT }}"           {{ lc }}  # [bla_vendor == 'Generic']
+      -DLAPACK_LIBRARIES="{{ PREFIX }}/lib/liblapacke{{ SHLIB_EXT }};{{ PREFIX }}/lib/liblapack{{ SHLIB_EXT }}"     {{ lc }}  # [bla_vendor == 'Generic']
```

Looks like following is needed. 
```suggestion
      -DLAPACK_LIBRARIES="{{ PREFIX }}/lib/liblapacke{{ SHLIB_EXT }};{{ PREFIX }}/lib/liblapack{{ SHLIB_EXT }};{{ PREFIX }}/lib/libcblas{{ SHLIB_EXT }};{{ PREFIX }}/lib/libblas{{ SHLIB_EXT }}"     {{ lc }}  # [bla_vendor == 'Generic']
```

---

i'm just following your lead :)